src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -851,6 +851,18 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            // apply http configuration configuration
+            configSection = (JObject)config["http"];
+            string routePrefix = ScriptConstants.DefaultHttpRoutePrefix;
+            if (configSection != null)
+            {
+                if (configSection.TryGetValue("routePrefix", out value))
+                {
+                    routePrefix = (string)value;
+                }
```

Comment:
The plan was to prevent any routes from using the "admin" prefix, as among other things, this could have potentially unintended consequences. 
Validation and early failure here would be a good idea. Perhaps some tests to make sure those routes are disallowed as well?